### ISSUE TITLE
[Site Request] www.skynews.com.au

### ISSUE THREAD

**Issue by nicolaasjan** [yt-dlp/yt-dlp#1287]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running yt-dlp version **2021.10.10**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] The provided URLs do not contain any DRM to the best of my knowledge
- [x] I've searched the bugtracker for similar site support requests including closed ones
- [x] I've read the opening an issue section in CONTRIBUTING.md
- [x] I have given an appropriate title to the issue

## Example URLs

- Single video: https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71

## Description

When trying to download the video linked above, I got this (Unsupported URL):

```python
yt-dlp -v https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71
[debug] Command-line config: ['-v', 'https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71']
[debug] User config file: /home/nico/.config/yt-dlp/config
[debug] User config: ['-o', '/dev/shm/test-dlp/%(title)s.%(ext)s', '-f', 'bestvideo[height<=1080][ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]/best', '--no-mtime', '--add-metadata', '--embed-thumbnail', '--force-ipv4', '--sponsorblock-remove', 'all', '--downloader', 'aria2c', '--downloader-args', 'aria2c:-x 10 -s 10 -j 10 -k 1M --log-level=info --file-allocation=none']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] yt-dlp version 2021.10.10 (zip)
[debug] Lazy loading extractors enabled
[debug] Python version 3.6.9 (CPython 64bit) - Linux-5.4.0-87-generic-x86_64-with-LinuxMint-19.3-tricia
[debug] exe versions: ffmpeg N-104315-g148ada5577-Nico-20211011-With-VP9-Timestamp-Fix, ffprobe N-104315-g148ada5577-Nico-20211011-With-VP9-Timestamp-Fix
[debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite, websockets
[debug] ANSI escape support: stdout = True, stderr = True
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71
[generic] 0f4c6243d6903502c01251f228b91a71: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] 0f4c6243d6903502c01251f228b91a71: Downloading webpage
[generic] 0f4c6243d6903502c01251f228b91a71: Extracting information
ERROR: [generic] Unsupported URL: https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71
Traceback (most recent call last):
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 590, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/generic.py", line 3706, in _real_extract
    raise UnsupportedError(url)
yt_dlp.utils.UnsupportedError: Unsupported URL: https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71
Traceback (most recent call last):
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 590, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/generic.py", line 3706, in _real_extract
    raise UnsupportedError(url)
yt_dlp.utils.UnsupportedError: Unsupported URL: https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1261, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1286, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 607, in extract
    e.msg, video_id=video_id, ie=self.IE_NAME, tb=e.traceback, expected=e.expected, cause=e.cause)
yt_dlp.utils.ExtractorError: [generic] Unsupported URL: https://www.skynews.com.au/world-news/united-states/incredible-vision-shows-lava-overflowing-from-spains-la-palma-volcano/video/0f4c6243d6903502c01251f228b91a71

```

Or is DRM involved here?
[Edit]
No DRM.

**Comment by nicolaasjan** [comment]
I see now that I can grab the m3u8 with the Stream Detector extension and when giving that to yt-dlp it downloads (with a generic file name):

```python
nico@nico-desktop:~$ dlp -v https://manifest.prod.boltdns.net/manifest/v1/hls/v4/clear/5647924203001/91227fd9-8e4c-4a1e-a34c-bd13e36abe93/67a11c75-e0b3-4b48-bd4f-54d039e0d4a4/10s/rendition.m3u8?fastly_token=NjE4ZTlmZTJfNWI5ZTIyNWYyMDZhODA4OTFiMTlkZmYwYzViYWRjMzEzMDZlNDJmMzg4NWUzMzNlZTAzN2RiODE4NzA0Zjg3Nw%3D%3D&pubid=5348771529001
[1] 17118
nico@nico-desktop:~$ [debug] Command-line config: ['-v', 'https://manifest.prod.boltdns.net/manifest/v1/hls/v4/clear/5647924203001/91227fd9-8e4c-4a1e-a34c-bd13e36abe93/67a11c75-e0b3-4b48-bd4f-54d039e0d4a4/10s/rendition.m3u8?fastly_token=NjE4ZTlmZTJfNWI5ZTIyNWYyMDZhODA4OTFiMTlkZmYwYzViYWRjMzEzMDZlNDJmMzg4NWUzMzNlZTAzN2RiODE4NzA0Zjg3Nw%3D%3D']
[debug] User config file: /home/nico/.config/yt-dlp/config
[debug] User config: ['-o', '/dev/shm/test-dlp/%(title)s.%(ext)s', '-f', 'bestvideo[height<=1080][ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]/best', '--no-mtime', '--add-metadata', '--embed-thumbnail', '--force-ipv4', '--sponsorblock-remove', 'all', '--downloader', 'aria2c', '--downloader-args', 'aria2c:-x 10 -s 10 -j 10 -k 1M --log-level=info --file-allocation=none']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] yt-dlp version 2021.10.10 (zip)
[debug] Lazy loading extractors enabled
[debug] Python version 3.6.9 (CPython 64bit) - Linux-5.4.0-87-generic-x86_64-with-LinuxMint-19.3-tricia
[debug] exe versions: ffmpeg N-104315-g148ada5577-Nico-20211011-With-VP9-Timestamp-Fix, ffprobe N-104315-g148ada5577-Nico-20211011-With-VP9-Timestamp-Fix
[debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite, websockets
[debug] ANSI escape support: stdout = True, stderr = True
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://manifest.prod.boltdns.net/manifest/v1/hls/v4/clear/5647924203001/91227fd9-8e4c-4a1e-a34c-bd13e36abe93/67a11c75-e0b3-4b48-bd4f-54d039e0d4a4/10s/rendition.m3u8?fastly_token=NjE4ZTlmZTJfNWI5ZTIyNWYyMDZhODA4OTFiMTlkZmYwYzViYWRjMzEzMDZlNDJmMzg4NWUzMzNlZTAzN2RiODE4NzA0Zjg3Nw%3D%3D
[generic] rendition: Requesting header
[generic] rendition: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[SponsorBlock] SponsorBlock is not supported for Generic
[info] rendition: Downloading 1 format(s): 0
[debug] Invoking downloader on "https://manifest.prod.boltdns.net/manifest/v1/hls/v4/clear/5647924203001/91227fd9-8e4c-4a1e-a34c-bd13e36abe93/67a11c75-e0b3-4b48-bd4f-54d039e0d4a4/10s/rendition.m3u8?fastly_token=NjE4ZTlmZTJfNWI5ZTIyNWYyMDZhODA4OTFiMTlkZmYwYzViYWRjMzEzMDZlNDJmMzg4NWUzMzNlZTAzN2RiODE4NzA0Zjg3Nw%3D%3D"
[hlsnative] Downloading m3u8 manifest
[hlsnative] Fragment downloads will be delegated to aria2c
[hlsnative] Total fragments: 8
[download] Destination: /dev/shm/test-dlp/rendition.mp4
[debug] aria2c command line: aria2c -c --console-log-level=warn --summary-interval=0 --download-result=hide --file-allocation=none -x16 -j16 -s16 --allow-overwrite=true --allow-piece-length-change=true --header 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.39 Safari/537.36' --header 'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' --header 'Accept-Encoding: gzip, deflate' --header 'Accept-Language: en-us,en;q=0.5' --interface 0.0.0.0 --check-certificate=true --remote-time=false -x 10 -s 10 -j 10 -k 1M --log-level=info --file-allocation=none --dir /dev/shm/test-dlp/ --auto-file-renaming=false --file-allocation=none --uri-selector=inorder -i /dev/shm/test-dlp/rendition.mp4.part.frag.urls
[aria2c] Downloaded 19124864 bytes90%)][#586cad 2.6MiB/2.7MiB(98%)]            
[download] 100% of 18.24MiB in 00:04
[debug] ffmpeg command line: ffprobe -show_streams file:/dev/shm/test-dlp/rendition.mp4
[ModifyChapters] SponsorBlock information is unavailable
[Metadata] Adding metadata to "/dev/shm/test-dlp/rendition.mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i file:/dev/shm/test-dlp/rendition.mp4 -map 0 -dn -c copy -metadata title=rendition -metadata 'purl=https://manifest.prod.boltdns.net/manifest/v1/hls/v4/clear/5647924203001/91227fd9-8e4c-4a1e-a34c-bd13e36abe93/67a11c75-e0b3-4b48-bd4f-54d039e0d4a4/10s/rendition.m3u8?fastly_token=NjE4ZTlmZTJfNWI5ZTIyNWYyMDZhODA4OTFiMTlkZmYwYzViYWRjMzEzMDZlNDJmMzg4NWUzMzNlZTAzN2RiODE4NzA0Zjg3Nw%3D%3D' -metadata 'comment=https://manifest.prod.boltdns.net/manifest/v1/hls/v4/clear/5647924203001/91227fd9-8e4c-4a1e-a34c-bd13e36abe93/67a11c75-e0b3-4b48-bd4f-54d039e0d4a4/10s/rendition.m3u8?fastly_token=NjE4ZTlmZTJfNWI5ZTIyNWYyMDZhODA4OTFiMTlkZmYwYzViYWRjMzEzMDZlNDJmMzg4NWUzMzNlZTAzN2RiODE4NzA0Zjg3Nw%3D%3D' file:/dev/shm/test-dlp/rendition.temp.mp4
[EmbedThumbnail] There aren't any thumbnails to embed

```

**Comment by nicolaasjan** [comment]
Thanks!
Worked well.
